packages/vats/src/core/types.js

```diff
@@ -119,52 +150,91 @@
  *     getRUNBundle: Producer<{ moduleFormat: string }>,
  *     vaultBundles: Producer<Record<string, { moduleFormat: string }>>,
  *     governanceBundles: Producer<Record<string, { moduleFormat: string }>>,
+ *     initialSupply: Producer<Payment>,
+ *     centralSupplyBundle: Producer<SourceBundle>,
  *     feeMintAccess: Producer<FeeMintAccess>,
  *     nameAdmins: Producer<Store<NameHub, NameAdmin>>,
+ *     priceAuthorityVat: Producer<PriceAuthorityVat>,
  *     priceAuthority: Producer<PriceAuthority>,
  *     priceAuthorityAdmin: Producer<PriceAuthorityRegistryAdmin>,
- *     vaultFactoryCreator: Producer<unknown>,
+ *     pegasusBundle: Producer<SourceBundle>,
+ *     vaultFactoryCreator: Producer<{ makeCollectFeesInvitation: () => Promise<Invitation> }>,
  *     vaultFactoryGovernorCreator: Producer<unknown>,
  *     vaultFactoryVoteCreator: Producer<unknown>,
  *     zoe: Producer<ERef<ZoeService>>,
  *   },
  * }} EconomyBootstrapPowers
  *
+ * IDEA/TODO: make types of demo stuff invisible in production behaviors
  * @typedef {{
- *   devices: {
- *      timer: unknown,
- *      bridge: Device<import('../bridge.js').BridgeDevice>,
- *      vatAdmin: unknown,
- *   },
- *   vats: {
- *     comms: CommsVatRoot,
- *     timer: TimerVat,
- *     vattp: VattpVat,
- *     vatAdmin: VatAdminVat,
+ *   argv: {
+ *     ROLE: string,
+ *     hardcodedClientAddresses: string[],
+ *     noFakeCurrencies: boolean,
+ *     FIXME_GCI: string,
```

Comment:
is this FIXME a merge blocker?

Reply:
it hasn't been for some number of years :)
again, this code is copied / moved from `bootstrap.js`.